### ISSUE TITLE
scheduler does not start with "enable_environment_macros=1"

### ISSUE THREAD

**Issue by spea1** [Alignak-monitoring/alignak#845]
scheduler does not start with "enable_environment_macros=1" in alignak.cfg

````
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler]  -> : worker zombie (now)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler] Action: c51a0d800b4b42c5913f730b9fe662a7 (None / fork)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler]  -> : worker zombie (now)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler] Action: cbcfb035e56e4e2c83bed34cf4331606 (None / fork)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler]  -> : worker zombie (now)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler] Action: edb5eed248f248ab825837d83f6aaa45 (None / fork)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler]  -> : worker zombie (now)
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler] Action: 749ba493e7834aa7a91a305c85cb91a9 (None / fork)
[2017-06-01 12:37:44 UTC] CRITICAL: [alignak.daemon] I got an unrecoverable error. I have to exit.
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler]  -> : worker zombie (now)
[2017-06-01 12:37:44 UTC] CRITICAL: [alignak.daemon] You can get help at https://github.com/Alignak-monitoring/alignak
[2017-06-01 12:37:44 UTC] DEBUG: [alignak.scheduler] Action: 810eaf5d0e3947f58fae921421890f74 (None / fork)
[2017-06-01 12:37:44 UTC] CRITICAL: [alignak.daemon] If you think this is a bug, create a new issue including as much detail
[2017-06-01 12:37:44 UTC] CRITICAL: [alignak.daemon] -----
[2017-06-01 12:37:44 UTC] CRITICAL: [alignak.daemon] Back trace of the error: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/schedulerdaemon.py", line 411, in main
    self.do_mainloop()
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemon.py", line 388, in do_mainloop
    self.do_loop_turn()
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/schedulerdaemon.py", line 229, in do_loop_turn
    self.sched.run()
  File "/usr/local/lib/python2.7/dist-packages/alignak/scheduler.py", line 2082, in run
    self.schedule()
  File "/usr/local/lib/python2.7/dist-packages/alignak/scheduler.py", line 1758, in schedule
    self.macromodulations, self.checkmodulations, self.checks))
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/schedulingitem.py", line 1256, in schedule
    checkmodulations, checks, force=force)
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/schedulingitem.py", line 2353, in launch_check
    env = macroresolver.get_env_macros(macrodata)
  File "/usr/local/lib/python2.7/dist-packages/alignak/macroresolver.py", line 268, in get_env_macros
    value = self._get_value_from_element(obj, prop)
  File "/usr/local/lib/python2.7/dist-packages/alignak/macroresolver.py", line 214, in _get_value_from_element
    return unicode(value(*real_args))
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/host.py", line 1130, in get_total_services_warning
    return self._tot_services_by_state(services, 1)
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/host.py", line 1107, in _tot_services_by_state
    return str(sum(1 for s in self.services
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/host.py", line 1108, in <genexpr>
    if services[s].state_id == state))
  File "/usr/local/lib/python2.7/dist-packages/alignak/objects/item.py", line 997, in __getitem__
    return self.items[key] if key else None
KeyError: u'e5f88a0249b44af1959b63ab7291e949'
````

**Comment by spea1** [comment]
Error found.
In "alignak/arbiter/daemons/arbiter-master.cfg" uncomment lines "timeout" and "data_timeout"

````
...
## Optional parameters:
## Uncomment these lines in a HA architecture so the master and slaves know
## how long they may wait for each other.
timeout                 3   ; Ping timeout
data_timeout            120 ; Data send timeout
#max_check_attempts      3   ; If ping fails N or more, then the node is dead
#check_interval          60  ; Ping node every N seconds
...
````

**Comment by mohierf** [comment]
This will probably be fixed with this PR: #819. To be confirmed when it is merged ...
